### DISCUSSION TITLE
Kick Alerts HELP!!!!

### DISCUSSION THREAD

**Discussion by Chico3tm** [Sehelitar/Kick.bot#9] [Q&A] [answered]
I'm very new to all this and need some assistance on understanding what I'm doing and how to do it. I'm trying to figure out the Alerts. I have figured out how to trigger obs chat commands. i just need help understanding the alerts for new followers, subs, gifts, host ect. I've gone over every doc in this and I can not make sense of it. If some have a example of a alert that would be great.

**Comment by Chico3tm** [comment]
So I've manage to figure out commands and alerts. I just need help with a auto thank you message for follow, sub, gifted ect. So like if I get a follow I would like to have a message pop up in chat thats say "Thanks %user% for the follow".

**Comment by Sehelitar** [comment, marked as answer]
Hey 👋
To send a message, just set the proper arguments and call the corresponding method.
And don't forget to add your trigger.
You should end up with something like this:
<img width="301" alt="image" src="https://github.com/Sehelitar/Kick.bot/assets/1716704/5b429123-da4e-464c-97be-7855e408452e">

**Reply by Chico3tm** [reply]
Thanks so much for that. I definitely needed that help, I've been working on this for days lol.
